simul/cothority/cothority.go

```diff
@@ -156,11 +153,7 @@ func main() {
 		allClosed <- true
 	}()
 	dbg.Lvl3(hostAddress, scs[0].Host.Entity.First(), "is waiting for all hosts to close")
-	select {
-	case <-allClosed:
-		dbg.Lvl2(hostAddress, ": all hosts closed")
-	case <-time.After(time.Second * time.Duration(scs[0].GetCloseWait())):
-		dbg.Lvl2(hostAddress, ": didn't close after", scs[0].GetCloseWait(), " seconds")
-	}
+	<-allClosed
```

Comment:
Because the `simul.go` has a timeout, and I didn't want to multiply timeouts. If the host doesn't close, `simul.go` will do a `Cleanup` of the platform. This way we only have two timeouts, as seen in `simul/runfiles/README.md`. If we add a timeout here, there would be overlapping timeouts.